src/components/CustomFeesEIP1559.vue

```diff
@@ -3,7 +3,7 @@
     <NavBar :showBackButton="false" :showBack="false" :showMenuList="false">
       Network Speed/Fee
     </NavBar>
-    <div class="wrapper form">
+    <div class="wrapper form" :class="{ wrapperPromiseSend: promiseSendView }">
```

Comment:
`promiseSendView`
that seems like a very weird name when what it does it add some padding. Can't you call it something more relevant? even sometihng like `padLabels`

I don't know the specifics but it doesn't seem appropriate for this component to look different based on where its embedded...

Reply:
applied mate